compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/textfield/ClearFocusBox.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.mpp.demo.textfield
+
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalFocusManager
+
+/**
+ * This wrapper need to control focus behavior on iOS to hide the keyboard.
+ */
+@Composable
+fun ClearFocusBox(content: @Composable () -> Unit) {
```

Comment:
I'd suggest to create modifier function (something like `clearFocusOnTap`) and pass it as extra parameter to `Screen.Example` instead of adding actual layout code to existing one-line lambdas

Reply:
Modifier will not helps in this case. Because we need an external Box() function to wrap content and fill max size.
I don't want to add Box to generic Example, because it may be harder to reproduce some use cases for all other screens.

Reply:
I will merge this PR, but if we will find better solution, we can apply it in the future.